SYCL/ESIMD/api/functional/ctors/ctor_copy.cpp

```diff
@@ -11,16 +11,18 @@
 // The current "REQUIRES" should be replaced with "gpu" only as mentioned in
 // "XREQUIRES".
 // UNSUPPORTED: cuda, hip
-// XRUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
-// XRUN: %GPU_RUN_PLACEHOLDER %t.out
-// RUN: false
-// XFAIL: *
-// TODO Unexpected static_assert was retrieved while calling simd::copy_from()
-// function. The issue was created (https://github.com/intel/llvm/issues/5112)
-// and the test must be enabled when it is resolved.
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
 //
 // Test for esimd copy constructor.
 
+// The following issues for simd<T,32> observed:
+// - freeze with T in {char, unsigned char, singned char};
+// - runtime failure with T in {short, unsigned short},
+//   details: https://github.com/intel/llvm/issues/5245
+// TODO Remove once the freeze is fixed
+#define SKIP_VECTOR_LEN_32
```

Comment:
Please remove this section and remote the usage of SKIP_VECTOR_LEN_32. 
Per the latest updates - the issues/5245 is closed.

Reply:
Done. An issue gone with the latest compiler and runtime versions.